example-model/src/main/java/uk/gov/gchq/palisade/example/client/SimpleClient.java

```diff
@@ -83,16 +82,17 @@ private DataRequestResponse makeRequest(final String fileName, final String reso
         requireNonNull(response, "response");
 
         final List<Stream<T>> dataStreams = new ArrayList<>(response.getResources().size());
-        for (final Entry<LeafResource, ConnectionDetail> entry : response.getResources().entrySet()) {
-            final ConnectionDetail connectionDetail = entry.getValue();
+        for (final LeafResource resource : response.getResources()) {
+            final ConnectionDetail connectionDetail = resource.getConnectionDetail();
```

Comment:
This needs to be updated to use the Eureka client
```suggestion
            final ServiceInstance dataService = getServiceInstances("data-service").get(0);
```

Reply:
This is the only usage of the connectionDetail field in palisade, it points to the correct data-service (say, a specific HadoopDataService for hdfs:/... resources)
This should not be using eureka.

Having said that, it should be validated that the correct connectiondetail is resolved through eureka by the resource-service when the connectiondetail is attached

Reply:
This has been resolved in Palisade-services